.github/workflows/testing.yml

```diff
@@ -86,6 +102,13 @@ jobs:
     steps:
       - name: Clone the connector
         uses: actions/checkout@v2
+        # This is needed for pull_request_target because this event runs in the
+        # context of the base commit of the pull request. It works fine for
+        # workflow_dispatch because the default behavior is used if
+        # ref/repository are empty strings.
```

Comment:
Thanks for the explanation!

Nit: works fine for workflow_dispatch -> works fine for 'push' and 'workflow_dispatch' events.

Reply:
(I'm interested how the 'push' event works more than 'workflow_dispatch', that's why I highlighted it.)

Reply:
> Nit: works fine for workflow_dispatch -> works fine for 'push' and 'workflow_dispatch' events.

Thank you, updated.

Reply:
> (I'm interested how the 'push' event works more than 'workflow_dispatch', that's why I highlighted it.)

I guess that:
`ref: {{$non_existent_value}}` transforms to `ref:` and it's a default-non-assigned value.

Anyway it works and checkouts a right commit: [run](https://github.com/oleg-jukovec/go-tarantool/runs/6740807255) for a [commit with empty ref/repository](https://github.com/oleg-jukovec/go-tarantool/blob/b768b9882c138961c904b1fc2622e42813e987af/.github/workflows/testing.yml#L35-L39).